wasabi/tables.py

```diff
@@ -78,45 +79,45 @@ def table(
         "fg_colors": fg_colors,
         "bg_colors": bg_colors,
     }
-    divider_row = row(["-" * width for width in widths], **settings)
+    divider_row = row(["-" * width for width in widths], **settings)  # type: ignore
```

Comment:
I'm not familiar how to do good typing when using `**kwargs`. So for this I opted to just ignore the type.